docs/running-on-kubernetes.md

```diff
@@ -1083,14 +1083,6 @@ See the [configuration page](configuration.html) for information on Spark config
   </td>
   <td>2.4.0</td>
 </tr>
-<tr>
-  <td><code>spark.kubernetes.pyspark.pythonVersion</code></td>
-  <td><code>"3"</code></td>
-  <td>
-   This sets the major Python version of the docker image used to run the driver and executor containers. Can be 3.
-  </td>
-  <td>2.4.0</td>
-</tr>
```

Comment:
The disappearance of the `pythonVersion` might make end users a bit hard to follow quickly. Seems no migration guide is there for kubernetes. Should we mention the removing and the support of PYSPARK_PYTHON and PYSPARK_DRIVER_PYTHON env variables in this doc?

Reply:
`pythonVersion` should (imo) go through a standard deprecation cycle, even if it's technically a no-op. That should include documentation and `@deprecated`

Reply:
@erikerlandson, just to extra clarify, there's no need to `@deprecated`. The configuration instance is not an API. It's `private[spark]`. It would have been nicer to clarify what's the actual gain of deprecating over removing a no-op configuration.